packages/dosespot-react/esbuild.mjs

```diff
@@ -0,0 +1,46 @@
+/* global console */
+/* global process */
+/* eslint no-console: "off" */
+
+import esbuild from 'esbuild';
+import { writeFileSync } from 'fs';
+
+const options = {
+  entryPoints: ['./src/index.ts'],
+  bundle: true,
+  platform: 'browser',
+  loader: { '.ts': 'ts', '.tsx': 'tsx' },
+  logLevel: 'info',
+  resolveExtensions: ['.ts', '.tsx'],
+  target: 'es2021',
+  tsconfig: 'tsconfig.build.json',
+  minifyWhitespace: true,
+  minifyIdentifiers: false,
+  minifySyntax: true,
+  sourcemap: true,
+  external: ['@medplum/core', '@medplum/mock', '@medplum/react', 'react'],
```

Comment:
May need to add `@medplum/react-hooks` here?

This may be something we need to check in other things importing `@medplum/react` too

Reply:
Good call